src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -64,4 +81,60 @@ public Response getDatasetByIdentifier(@Auth AuthUser authUser, @PathParam("iden
             return createExceptionResponse(e);
         }
     }
+
+    @POST
+    @Consumes("application/json")
+    @Produces("application/json")
+    @Path("/dar/draft")
+    @RolesAllowed({ADMIN})
+    public Response createDraftDataAccessRequest(
+        @Auth AuthUser authUser,
+        @Context UriInfo info,
+        @QueryParam("identifiers") String identifiers,
+        @QueryParam("projectTitle") String projectTitle) {
+      try {
+        // Ensure that the user is a registered DUOS and SAM user
+        User user = findOrCreateUser(authUser);
+        if (Objects.isNull(identifiers) || identifiers.isBlank()) {
+          throw new BadRequestException("No dataset identifiers were provided");
+        } else {
+          List<Integer> datasetIds = Arrays.asList(identifiers.split(","))
+              .stream()
+              .filter(identifier -> !identifier.isBlank())
+              .map(identifier -> this.datasetService.findDatasetByIdentifier(identifier).getDataSetId())
+              .collect(Collectors.toList());
+        if (datasetIds.isEmpty()) {
+            throw new BadRequestException("No datasets associated with identifiers: " + identifiers);
+        }
```

Comment:
There is an error case here that we need to somehow deal with. If the user passed in 3 identifiers, and we only found 2 of them, we should error out in some way with a useful error message.

Reply:
Updated the failure logic to check that the length of the identifier list is equal to the length of the datasetId list we get back from the service method.